templates/web/common/includes/display_product_api.tt.html

```diff
@@ -0,0 +1,32 @@
+
+[% IF single_param("jqm") %]
+    [% Lang.app_please_take_pictures.$request_lc %]
+    <button onclick="captureImage();" data-icon="off-camera">[% Lang.app_take_a_picture.$request_lc %]</button>
+    <div id="upload_image_result"></div>
+    <p>[% Lang.app_take_a_picture_note.$request_lc %]</p>
+[% END %]
+
+[% IF api_version >= 0.1 %]
+    <form id="product_fields" action="javascript:void(0);">
+        <div data-role="fieldcontain" class="ui-hide-label" style="border-bottom-width: 0;">
+            [% FOREACH field IN app_fields %]
+                [% value = Lang.$field.$request_lc %]
+                <label for="[% field %]">[% Lang.$field.$request_lc %]</label>
```

Comment:
The Lang.$field.$request_lc does not work, because you forgot to pass %Lang as parameter !